Assets/Scripts/Interactables/Harvestable.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Inventory;
+using Managers;
+using Utils;
+
+namespace Interactables
+{
+    public class Harvestable : Interactable
+    {
+        [Tooltip("The items being dropped on a successful harvest.")]
+        [SerializeField] 
+        private List<Item> drops;
```

Comment:
```suggestion
        private List<ItemResourceData> drops;
```
This would combine an item and the amount. Not needed, just a hint:)